xml/System.IO/UnmanagedMemoryStream.xml

```diff
@@ -1574,10 +1574,10 @@
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="1" FrameworkAlternate="netcore-3.0" />
       </Parameters>
       <Docs>
-        <param name="buffer">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="buffer">Buffer that will be written.</param>
+        <param name="cancellationToken">Token that can be used to cancel the operation.</param>
+        <summary>Writes buffer into the stream. The operation completes synchronously.</summary>
```

Comment:
```suggestion
        <summary>Asynchronously writes the specified buffer into the current stream.</summary>
```

Reply:
Doesn't "The operation completes synchronously." makes this summary confusing for an async method?